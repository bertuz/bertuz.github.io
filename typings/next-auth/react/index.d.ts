import type {
  SessionProviderProps,
  UseSessionOptions,
} from 'next-auth/react/types';

import type { Session } from '../core';

import type { GetSessionParams, SessionContextValue } from 'next-auth/react';

export declare function getSession(
  params?: GetSessionParams
): Promise<Session | null>;

export declare function useSession<R extends boolean>(
  options?: UseSessionOptions<R>
): SessionContextValue<R>;

export declare function SessionProvider(
  props: SessionProviderProps
): JSX.Element;
