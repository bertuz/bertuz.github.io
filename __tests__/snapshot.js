import { render } from '@testing-library/react'
import Home from '../pages/index'

it('renders homepage unchanged', () => {
    const givenArticles = [];
    const { container } = render(<Home articles={givenArticles} />)
    expect(container).toMatchSnapshot()
})
