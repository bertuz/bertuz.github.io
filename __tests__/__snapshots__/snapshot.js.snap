// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1
      data-testid="t1"
    >
      Welcome to my page!
    </h1>
    <div
      class="grid"
    />
  </div>
</div>
`;
